### PR TITLE
Add convenience command to load all installed workflows

### PR DESCRIPTION
**Pull request by jrbotros**

### CODE REVIEW

File: example_project/example_project/orchestra_settings.py [hunk 1]
```diff
@@ -68,11 +68,11 @@ def setup_orchestra(settings_module_name):
     # Tasks and Workflows
     ######################
 
-    # Installed orchestra workflows
-    settings.ORCHESTRA_PATHS = (
-        ('simple_workflow.workflow', 'simple_workflow'),
-        ('journalism_workflow.workflow', 'journalism_workflow'),
+    # Installed orchestra workflows to be included as Django apps
```
Comment (marcua) [inline example_project/example_project/orchestra_settings.py:62]: do we need doc updates for this?

File: orchestra/workflow/load.py [hunk 2]
```diff
@@ -11,6 +12,20 @@
 from orchestra.core.errors import WorkflowError
 
 
+def get_workflow_version_slugs():
+    versions = {}
+    for app_name in settings.ORCHESTRA_WORKFLOWS:
+        # App label is the last part of the app name by default
+        app_label = app_name.split('.')[-1]
```
Comment (marcua) [inline orchestra/workflow/load.py:19]: if two apps have the same workflow slug, we should raise

then this thing would just return {workflow_slug: [version, slugs]}

File: orchestra/management/commands/loadallworkflows.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            '--force', action='store_true', default=False,
+            help=('If a workflow version already exists, update the existing '
+                  'version instead of exiting.'))
+
+    def handle(self, *args, **options):
+        workflow_version_slugs = get_workflow_version_slugs()
+        for app_label, workflow_info in workflow_version_slugs.items():
+            for version_slug in workflow_info['versions']:
+                try:
+                    print('Loading {} {} from {}'.format(
```
Comment (marcua) [inline orchestra/management/commands/loadallworkflows.py]: do we do self.stdout.write so it's easier to test?

Comment (jrbotros) [inline orchestra/management/commands/loadallworkflows.py]: :white_check_mark:

File: example_project/example_project/orchestra_settings.py (hunk 1 above)
Comment (jrbotros) [inline example_project/example_project/orchestra_settings.py:62]: I don't think we talk about the settings file anywhere other than in these comments—let me know if I'm wrong!

File: orchestra/workflow/load.py (hunk 2 above)
Comment (jrbotros) [inline orchestra/workflow/load.py:19]: :white_check_mark:

File: example_project/example_project/orchestra_settings.py (hunk 1 above)
Comment (marcua) [inline example_project/example_project/orchestra_settings.py:62]: Two quick ones hidden in http://orchestra.readthedocs.org/en/stable/getting_started.html#create-a-django-project
- "Make sure you added simple_workflow to your INSTALLED_APPS setting following the previous section."
- "Add 'simple_workflow' to settings.INSTALLED_APPS in the “General” section if you want to run the demo workflow (instructions below), and add 'journalism_workflow' if you want to run the journalism workflow."

**Comment by marcua** [comment]
lgtm w/ the two doc fixes---no need for more review!:)

File: example_project/example_project/orchestra_settings.py (hunk 1 above)
Comment (jrbotros) [inline example_project/example_project/orchestra_settings.py:62]: :white_check_mark:

**Pull request merged**